opentracing-util/src/test/java/io/opentracing/util/ThreadLocalScopeTest.java

```diff
@@ -21,49 +21,17 @@
 import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 
-import io.opentracing.ActiveSpan;
+import io.opentracing.Scope;
 import io.opentracing.Span;
 import org.junit.Before;
 import org.junit.Test;
 
-public class ThreadLocalActiveSpanTest {
-    private ThreadLocalActiveSpanSource source;
+public class ThreadLocalScopeTest {
+    private ThreadLocalScopeManager scopeManager;
 
     @Before
     public void before() throws Exception {
-        source = new ThreadLocalActiveSpanSource();
-    }
-
-    @Test
-    public void continuation() throws Exception {
```

Comment:
Although continuation not included, still need a test for when `finishOnClose` is false - so ensuring `finish` is not closed when the scope is closed.